src/components/Options.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import tw from "twin.macro";
+import { MaxWidthWrapper } from "./Wrappers";
+import KPIOptions from "./KPIOptions";
+import OptionsList from "./OptionsList";
+
+const Options: React.FC = () => {
+  return (
+    <MaxWidthWrapper>
+      <Wrapper>
+        <KPIOptions />
```

Comment:
+1 to @daywiss 's other comment about preferring to make components like `KpiOptions` stateless, would be great if you could initialize the containers in the higher level components like this `Options` one and pass them into stateless children components